tests/pytests/data/LUD3125I.161/LUD3125I.161_isis3.lbl

```diff
@@ -0,0 +1,140 @@
+PDS_VERSION_ID               = PDS3
```

Comment:
This needs to be the actual ISIS cube label instead of the lbl file from the PDS3 image. The easiest way to get the ISIS label is to run `catlab` on the ISIS cube then just overwrite this file